stage1/init/pod.go

```diff
@@ -249,7 +249,7 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, am *schema.ImageManifest, inte
 		}
 	}
 
-	for _, i := range am.App.Isolators {
+	for _, i := range app.Isolators {
```

Comment:
Hrm. This is sucky. What do you think about changing the signature of this function (and appToSystemdNspawnArgs) to accept an App instead?